bundles/CoreBundle/Command/Migrate/StorageCommand.php

```diff
@@ -26,7 +26,7 @@
 /**
  * @internal
  */
-class MigrateStorageCommand extends AbstractCommand
+class StorageCommand extends AbstractCommand
```

Comment:
This command is defined here: 
https://github.com/pimcore/pimcore/blob/417b50202ceba7721311eb73f92c308f84236546/bundles/CoreBundle/Resources/config/commands.yaml#L76

So this needs to be changes as well when changing the FQCN😉

Reply:
Done.